integration/pubsub_test.go

```diff
@@ -114,6 +117,30 @@ func (suite *serviceBusSuite) TestPublishAndListenRetryLater() {
 	}, event)
 }
 
+// Note that, this test need manually verify, you should be able to see 3 msgs are receievd in a row from the test log
```

Comment:
it's awesome to have the order verification, the comment here is outdated :D

Reply:
question, why do we have order verification?
servicebus can't guarantee order by design. especially with concurrent workers pulling from the subscription?

Reply:
Really we wanted to show that we recieved messages before we completed others (actually proved we were parallel) Showing the ordering was mostly for debugging the test could just be that we got a certain number of recieves before completes.